client/web/web.go

```diff
@@ -480,11 +480,13 @@ func (s *Server) serveAPIAuthSessionNew(w http.ResponseWriter, r *http.Request)
 		}
 		// Set the cookie on browser.
 		http.SetCookie(w, &http.Cookie{
-			Name:    sessionCookieName,
-			Value:   session.ID,
-			Raw:     session.ID,
-			Path:    "/",
-			Expires: session.expires(),
+			Name:     sessionCookieName,
+			Value:    session.ID,
+			Raw:      session.ID,
+			Path:     "/",
+			HttpOnly: true,
+			SameSite: http.SameSiteLaxMode,
```

Comment:
Can we try Strict? It seems to work for me in local testing. If Strict can't work, we should document why.

I also think it may be worth documenting why we don't set Secure, e.g.:

```go
+                       // We can't set this to true because we serve over HTTP (but only on
+                       // Tailscale IPs, hence over encrypted connections that a LAN-local
+                       // attacker cannot sniff). In the future, it might be good to
+                       // also/only serve via https://hostname.tailnet-name.ts.net; then we
+                       // could set this.
+                       //Secure:   true,
```

Reply:
yeah, I can't seem to get Strict to break either in local testing.  I was going off of what MDN said.  :shrug:

Switching to "String" and will add note about Secure